gluten-core/src/main/scala/io/glutenproject/expression/ExpressionConverter.scala

```diff
@@ -240,6 +240,7 @@ object ExpressionConverter extends Logging {
         new Murmur3HashTransformer(exprs, expr)
       case l: StringTrim =>
         if (l.trimStr != None) {
+          // todo to be remove when Velox support this argument
```

Comment:
it's better use: `// TODO: to be remove when Velox support this argument`

Reply:
Updated.